### PR TITLE
A little RN clean-up and adding a few KI from the GH tracker

### PR DESCRIPTION
**Pull request by opayne1**
A little clean-up and resolving some entries from the [GH issue tracker ](https://github.com/openshift/openshift-docs/issues/43249)

Link to docs preview:
(known issues section) http://file.rdu.redhat.com/opayne/RN-cleanup-2/release_notes/ocp-4-11-release-notes.html#ocp-4-11-known-issues

Additional information:
[2096496](https://bugzilla.redhat.com/show_bug.cgi?id=2096496) was removed from known issues section and added to the bug fix section based on [this comment to the GH issue tracker ](https://github.com/openshift/openshift-docs/issues/43249#issuecomment-1207950133)

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 == About this release
 
 // TODO: Update with the relevant information closer to release.
-{product-title} (link:https://access.redhat.com/errata/RHSA-2022:5069[RHSA-2022:5069]) is now available. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md[Kubernetes 1.24] with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} {product-version} are included in this topic.
+{product-title} (link:https://access.redhat.com/errata/RHSA-2022:5069[RHSA-2022:5069]) is now available. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md[Kubernetes 1.25] with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} {product-version} are included in this topic.
```
Comment (opayne1) [inline release_notes/ocp-4-11-release-notes.adoc:17]: I am not quite sure this is correct. Going to verify.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -1385,6 +1385,7 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 . When the vMedia attachment request fails with an error indicating that the `TransferProtocolType` attribute is missing, retry the request and explicitly specify this attribute.
 . Check for the presence of the RedFish settings resource in the system. If it is present, use it for the boot sequence override.
 
++
```
Comment (nalhadef) [inline release_notes/ocp-4-11-release-notes.adoc:1388]: Do you want the + to carry over the formatting of the numbered item preceeding ("Check for the presence..."). if so, I think you need to delete the blank line (1387) before the +. At the moment, no special formatting is carried over.

image

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -1585,6 +1586,8 @@ sourceStrategy:
 
 * Because vSphere UPI clusters do not set the `PlatformStatus.VSphere` parameter at installation, the parameter was set to `nil`. This caused the MCO logs to be filled with unnecessary and repetitive messages that this parameter cannot have the value `nil`. This fix removes the warning, which was added to resolve a separate issue. As a result, the logs no longer list this message for vSphere UPI installations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2080267[*BZ#2080267*])
 
+* Previously, when you attempted to create a cluster with both FIPS and `realTimeKernel` the Machine Config Operator (MCO) degraded due to a merge logic issue within the code. With this update, the MCO no longer degrades whne creating a cluster with both FIPS and `realTimeKernel`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2096496[*BZ#2096496*])
```
Comment (nalhadef) [inline release_notes/ocp-4-11-release-notes.adoc:1589]: Typo: "whne"

File: release_notes/ocp-4-11-release-notes.adoc [hunk 4]
```diff
@@ -1847,7 +1850,7 @@ With this update, the CMO now properly propagates the external labels that you c
...
+* {product-title} allows installation of vSphere CSI drivers shipped by either the upstream community or VMware. Although Red Hat does not support this driver, cluster administrators can still install and use it because it has more features than the vSphere CSI driver shipped by Red Hat. {product-title} can be upgraded to {product-version} with the upstream and VMware vSphere CSI driver, however, you will be warned about the presence of a third party CSI driver. In a future release {product-title}, you will be required to upgrade to the version of vSphere CSI shipped by Red Hat before upgrading to the next version of {product-title}. For more information, see (link:https://bugzilla.redhat.com/show_bug.cgi?id=2089419[*BZ#2089419*]) and (link:https://bugzilla.redhat.com/show_bug.cgi?id=2052071[*BZ#2052071*]).
```
Comment (nalhadef) [inline release_notes/ocp-4-11-release-notes.adoc:1853]: I would delete the sentence "In a future release..." It's future talk and who knows what will happen in the meantime. Plus the reader doesn't need to know this now.
https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future

File: release_notes/ocp-4-11-release-notes.adoc [hunk 5]
```diff
@@ -2325,12 +2328,9 @@ This script removes unauthenticated subjects from the following cluster role bin
...
 
-* In the {rh-openstack-first}, a potential issue can affect Kuryr when the ports pools are populated with a variety of bulk port creation requests made at the same time. During these bulk requests, if the IP allocation for one of the IP addresses fails, the Neutron retries the operation for all ports. This issue was resolved in a previous errata; however, Red Hat suggests setting a small value for the ports pools batch to avoid large bulk port creation requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024690[*BZ2024690*])
+* In the {rh-openstack-first}, a potential issue can affect Kuryr when the ports pools are populated with a variety of bulk port creation requests made at the same time. During these bulk requests, if the IP allocation for one of the IP addresses fails, the Neutron retries the operation for all ports. This issue was resolved in a previous errata; however, Red Hat suggests setting a small value for the ports pools batch to avoid large bulk port creation requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024690[*BZ#2024690*])
```
Comment (nalhadef) [inline release_notes/ocp-4-11-release-notes.adoc:2333]: Recommend and its cousin suggest are frowned upon.
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r
https://redhat-documentation.github.io/supplementary-style-guide/#_r

File: release_notes/ocp-4-11-release-notes.adoc [hunk 6]
```diff
@@ -2422,23 +2422,41 @@ openshift.io/scc: hostmount-anyuid
...
+* When `resourceGroupID` is specified in `install-config.yaml`, an error is displayed when deleting bootstrap resources and {product-title} installation fails. As a workaround for this issue, do not specify `resourceGroupID` in `install-config.yaml`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100746[*BZ#2100746*])
+
+* There is a known issue with scale-up on RHEL compute nodes. The new nodes could turn into `Ready`, but Ingress pods cannot turn into `Running` on these nodes, and scale-up does not succeed. As a workaround, scale-up with {op-system} nodes does work. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2056387[*BZ#2056387*])
+
+* After creating a `machineset` in a Google Cloud Platform (GCP), the `capg-controller-manager` machine is stuck in `Provisioning`. There is currently no workaround for this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2107999[*BZ#2107999*])
+
+* There is a known issue on nutanix where persistent volumes (PVs) created by a cluster are not cleaned up by the  `destroy cluster` command. As a workaround for this issue, you need to clean up the PVs manually. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2108700[*BZ#2108700*])
```
Comment (nalhadef) [inline release_notes/ocp-4-11-release-notes.adoc:2437]: The first letter of "Nutanix" should be upper case.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (opayne1) [reply release_notes/ocp-4-11-release-notes.adoc:1388]: I am going to adjust the wording of that last sentence. I want the formatting from the bullet carried over, not to the numbered items.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (opayne1) [reply release_notes/ocp-4-11-release-notes.adoc:17]: Confirmed v1.24 from what's new in Openshift 4.11 slides

**Pull request merged**